packages/framework-provider-azure-infrastructure/src/infrastructure/stacks/event-handler-function.ts

```diff
@@ -0,0 +1,28 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { EventHandlerFunctionDefinition } from '../types/functionDefinition'
+
+export class EventHandlerFunction {
+  public constructor(readonly config: BoosterConfig) {}
+
+  public getFunctionDefinition(): EventHandlerFunctionDefinition {
+    return {
+      name: 'eventHandler',
+      config: {
+        bindings: [
+          {
+            type: 'cosmosDBTrigger',
+            name: 'rawEvent',
+            direction: 'in',
+            leaseCollectionName: 'leases',
+            connectionStringSetting: 'COSMOSDB_CONNECTION_STRING',
+            databaseName: this.config.resourceNames.applicationStack,
+            collectionName: this.config.resourceNames.eventsStore,
+            createLeaseCollectionIfNotExists: 'true',
+          },
+        ],
+        scriptFile: '../dist/index.js',
```

Comment:
I would say using `this.config.userProjectRootPath` would be safer. Would it be possible to use it here? (and in the other places where this string is used)

Reply:
Correct me if I'm wrong but this property returns the full project path. Example: `/Users/gonzalogarcia/dev/src/theam/booster_projects/booster-az-cart-demo/.deploy/`. Am I using it wrongly?